lean/src/valid.lean

```diff
@@ -852,25 +854,25 @@ begin
 end
 
 theorem amc12b_2021_p21
-  (h₀ : fintype {x : ℝ | 0 < x ∧ x^((2:ℝ)^real.sqrt 2) = (real.sqrt 2)^((2:ℝ)^x)}) :
-  ↑2 ≤ ∑ k in {x : ℝ | 0 < x ∧ x^((2:ℝ)^real.sqrt 2) = (real.sqrt 2)^((2:ℝ)^x)}.to_finset, k ∧ ∑ k in {x : ℝ | 0 < x ∧ x^((2:ℝ)^real.sqrt 2) = (real.sqrt 2)^((2:ℝ)^x)}.to_finset, k < 6 :=
+  (h₀ : fintype {x : ℝ | 0 < x ∧ x^((2 : ℝ)^real.sqrt 2) = (real.sqrt 2)^((2 : ℝ)^x)}) :
+  ↑2 ≤ ∑ k in {x : ℝ | 0 < x ∧ x^((2 : ℝ)^real.sqrt 2) = (real.sqrt 2)^((2 : ℝ)^x)}.to_finset, k ∧
+  ∑ k in {x : ℝ | 0 < x ∧ x^((2 : ℝ)^real.sqrt 2) = (real.sqrt 2)^((2 : ℝ)^x)}.to_finset, k < 6 :=
```

Comment:
you can rewrite this one the same way as `mathd_algebra_405`.

Reply:
It is not clear if writing it the same way as `mathd_algebra_405` is better, it seems to be the same thing for Lean but maybe it is simpler for us to read the statement when written as `mathd_algebra_405`

Reply:
oh then it's fine for me.